### ISSUE TITLE
trouble with avatars

### ISSUE THREAD

**Issue by bbrendon** [monicahq/monica#3387]
I'm having issues with Avatars....

First, I can't get any new avatars to be created. I have 12 avatars out of about 500 and I'm not really sure how the 12 got there. 

To solve the issue I have tried .
`docker-compose exec monicahq php artisan monica:update`

`docker-compose exec monicahq php artisan schedule:run`

This is the queue setting I'm using so I don't think I need a cronjob for it.
```
CACHE_DRIVER=database
SESSION_DRIVER=file
SESSION_LIFETIME=120
QUEUE_CONNECTION=sync
```

I was finally able to get a new avatar to appear in `app/public/avatars`, but only by creating a new contact. Changing the avatar settings on existing contacts between Adorable to Default seem to have no benefit for me.

I also tried using "Adorable" avatar on exiting contacts but that causes the contact avatar to look like a broken image in the contact list. 

It seems like I must be missing something simple.

**Comment by elanosilveira** [comment]
If you could share more details, such as the version you're using, it would help a ton.

**Comment by bbrendon** [comment]
Docker     image: monicahq/monicahq
v2.15.2

**Comment by bbrendon** [comment]
FYI. v2.16.0 appears to behave exactly the same

**Comment by asbiin** [comment]
Hi @bbrendon sorry for the situation.
Did you figured out how to have the avatars ?

**Comment by bbrendon** [comment]
No, I've just pretty much just gotten used to the problem and I've put pictures in the contacts I really care about.

**Comment by AlphaJack** [comment]
I'm having a similar problem, default and adorable avatars have an empty src and photos uploaded by me return "Internal server error 500". Files are correctly uploaded to `/usr/share/webapps/monica/storage/app/public/photos` with correct permissions/owner
![avatar](https://user-images.githubusercontent.com/19962243/89656553-46692000-d8bb-11ea-95bb-cd53663d0eb1.png)

**Comment by cashweaver** [comment]
> I'm having a similar problem, default and adorable avatars have an empty src and photos uploaded by me return "Internal server error 500". Files are correctly uploaded to `/usr/share/webapps/monica/storage/app/public/photos` with correct permissions/owner ![avatar](https://user-images.githubusercontent.com/19962243/89656553-46692000-d8bb-11ea-95bb-cd53663d0eb1.png)

I'm having a similar issue with photos. I'm running the docker fpm/supervisor image and am getting a 500 error for all photos I've uploaded.
